pkg/controllers/reverse_ip.go

```diff
@@ -44,6 +109,35 @@ func (c *NodeController) syncReverseIP(nodeName string) error {
 		return nil
 	}
 
+	if c.scwZoneFound {
+		comment := fmt.Sprintf("k8s node %s", nodeName)
+		_, err := dnsAPI.UpdateDNSZoneRecords(&dns.UpdateDNSZoneRecordsRequest{
+			DNSZone: c.reverseIPDomain,
```

Comment:
```suggestion
			DNSZone: c.scwZone,
```
no?

Reply:
it is tricky
when you want to fill records on tools.test.com
* you can have only test.com 
* you can have test.com and tools.test.com
* you can have only tools.test.com

when updating , we asked set it to tools.test.com
the API will "relocated" to the "nearest" parent
if name is 4-3-2-1 zone tools.test.com
it can transform it to  4-3-2-1.tools if need

if we use c.scwZone as dnsZone , we need to "relocated" here but since the API doest that ... :)

Reply:
i will use FQDN to use the same zone 
let me update / test that :)

Reply:
should be better :)